app/views/entities/new.html.erb

```diff
@@ -0,0 +1,33 @@
+<%= form_with(model: @entity, url: category_entities_path) do |form| %>
+    <div class = "app_header">
+        <a href ="<%= category_entities_path %>">
+            <i class="fa-solid fa-arrow-left" id="left-arrow"></i>
+        </a>
+        <p> New Transaction </p>
+        <a href="/logout" id = "logout" >
+            Logout
+        </a>
+    </div>
+
+    <div class = "form-container">
+        <div class="field">
+            <%= form.text_field :name, placeholder: "Name "%>
+            <% if @entity.errors.any? %>
+                <span class="alert">
+                    <%= @entity.errors[:name].first %>
+                </span>
+            <% end %>
+        </div>
+
+        <div class="field">
+            <%= form.text_field :amount, placeholder: "Price "%>
+            <% if @entity.errors.any? %>
+                <span class="alert">
+                    <%= @entity.errors[:amount].first %>
+                </span>
+            <% end %>
+        </div>
+
+        <%= form.submit "Save", class: "submit-button" %>
+    </div>
+<% end %>
```

Comment:
- [ ] As required the user should fill out a form to create a new transaction with:
name (mandatory)
amount (mandatory)
categories (mandatory at least one)

At the moment only two fields are available because the category is being added automatically, kindly fix this.

Your form
![image](https://github.com/Microverse-Fullstack-Program/rails-budget-app-v2/assets/81932478/77476af2-f18c-40a4-a419-074501844135)